app/components/layout/pinned-entity-dropdown/pinned-dropdown.component.ts

```diff
@@ -4,7 +4,9 @@ import { List } from "immutable";
 import { Observable, Subscription } from "rxjs";
 
 import { PinnableEntity, PinnedEntityType } from "@batch-flask/core";
+import { MouseButton } from "@batch-flask/core";
```

Comment:
These 2 imports should be combined into 1.

import { PinnableEntity, PinnedEntityType } from "@batch-flask/core";
import { MouseButton } from "@batch-flask/core";